src/hub.html

```diff
@@ -17,6 +19,28 @@
     <% } else { %>
         <script src="https://cdn.rawgit.com/brianpeiris/aframe/845825ae694449524c185c44a314d361eead4680/dist/aframe-master.js"></script>
     <% } %>
```

Comment:
We actually inject these tags server-side, since the contents are dynamic (right now, includes the name of the room) see: https://github.com/mozilla/reticulum/pull/23

We can update the reticulum code for any missing tags you've included here. Eventually some of these (like the thumbnail) will be informed by the current state of the room, such as the environment choice and number of CCU.

Reply:
Also worth noting that we plan on ripping out almost everything from this file and having it be rendered client side with react, at which point this will be just a stub and reticulum will have a proper templated server side view for the whole page.

Reply:
oh thanks! I was wondering where; this makes more sense!

it's useful to have the HTML document's skeleton (and ideally critical-path assets, like inlined CSS and minimal JS to make the page interactive).

what about server-side rendering with React? just curious about the desire to move more to the client.

Reply:
I'm not sure what's involved to do that with Phoenix, my guess is that it is probably prohibitive since this is running on the BEAM VM. I agree that would be a good approach if we were running on a JS VM though.

Reply:
Oh, and the desire to move it all into the client is basically just to decouple our reticulum build/deploy process from the hubs project's build process. Once this is done there's a straight shot to being able to a) remove the coupling, so builds and deploys are faster and b) it cleans up the self-hosting story, since reticulum right now is coupled with the "burned in" asset URLs in the client code at package build time, so we can't just tell people to run our Habitat package to get things working.

This is being tracked here: https://github.com/mozilla/hubs/issues/347